src/dune_rules/melange/melange.ml

```diff
@@ -2,13 +2,13 @@ open Import
 
 module Module_system = struct
   type t =
-    | Es6
+    | ESM
```

Comment:
OCD note: why `ESM` but not `CommonJS` nor `ES6`? 😄 
```suggestion
    | Esm
```

Reply:
I agree we should use `CommonJS` instead, but this page uses `ESM` all over the place https://nodejs.org/api/esm.html#modules-ecmascript-modules